cmd/finch/devcontainer_patch.go

```diff
@@ -108,3 +109,14 @@ func inspectContainerOutputHandler(cmd command.Command) error {
 	prettyPrintJSON(stdoutBuf.String())
 	return err
 }
+
+func handleDockerCompatComposeVersion(cmd command.Command) error {
+	logrus.Warn("Displaying a corresponding docker version set as environment variable DOCKER_COMPOSE_VERSION...")
+	ver := os.Getenv("DOCKER_COMPOSE_VERSION")
```

Comment:
We should be using `stdlib.Env` function here for easier mocking

Reply:
ok will do that

Reply:
is the package named exactly like that, i dont seem to be able to import this package.